cmd/sos_acl.go

```diff
@@ -76,27 +78,27 @@ var sosAddACLCmd = &cobra.Command{
 			return err
 		}
 
-		objInfo, err := minioClient.GetObjectACL(args[0], args[1])
+		objInfo, err := minioClient.GetObjectACLWithContext(gContext, bucketName, objectName)
 		if err != nil {
 			return err
 		}
 
 		objInfo.Metadata.Add("content-type", objInfo.ContentType)
 
-		src := minio.NewSourceInfo(args[0], args[1], nil)
+		src := minio.NewSourceInfo(bucketName, objectName, nil)
 
-		_, okMeta := meta["X-Amz-Acl"]
-		_, okHeader := objInfo.Metadata["X-Amz-Acl"]
+		_, hasNewAmzACL := meta["X-Amz-Acl"]
+		_, hasAmzACL := objInfo.Metadata["X-Amz-Acl"]
```

Comment:
Please comment what those variables are needed for: 2 variables with one named "new" is not enough to understand what is going on here.

Reply:
👍